lib/rules/flag-role.json

```
@@ -0,0 +1,13 @@
+{
+	"id": "flag-role",
+	"selector": "[role=\"none\"], [role=\"presentation\"]",
+	"tags": ["cat.aria"],
```

Please add the `best-practice` tag.
```suggestion
	"tags": ["cat.aria", "best-practice"],
```